UPGRADE.md

```diff
@@ -1503,6 +1503,56 @@ ALTER TABLE ca_category_translation_medias ADD CONSTRAINT FK_39FC41BA17CA14DA FO
 ALTER TABLE ca_category_translation_medias ADD CONSTRAINT FK_39FC41BA7DE8E211 FOREIGN KEY (idMedia) REFERENCES me_media (id) ON DELETE CASCADE;
 ```
 
+In addition that also the PHPCR tables have to be changed to utf8mb4 in case jackalope-doctrine-dbal is used:
+
+```sql
+ALTER TABLE `phpcr_binarydata` CHARACTER SET = utf8mb4;
```

Comment:
The above queries don't switch tables to utf8mb4 but to beneath does so after doing this you have mixed tables utf8 in the sulu tables and utf8mb4 in the phpcr nodes table so this is not correctk to set the character set here but not for the sulu tables.

Reply:
The thing is that if you are updating your schema when using the new `sulu-minimal`, then the all the tables generated by the ORM will be updated, but the jackalope tables will be untouched. That's also a strange behavior... I think we have to discuss this further.

Reply:
When thinking about it, I would say we should add these statements for the above tables as well... If the developer would not switch to utf8mb4, then the above lines would also not really be needed...

Reply:
They are needed because our schema is defined so. Else it will not be a validate schema. 
The different between sulu tables and phpcr is that sulu in general uses shorter characters and phpcr used only shorted characters when using utf8mb4.

Reply:
I know this difference, but according to this [great article about converting to `utf8mb4`](https://mathiasbynens.be/notes/mysql-utf8mb4) using just `utf8` also opens us up for some security vulnerabilities. So I would suggest we add the charset for all the tables to the `UPGRADE.md` as well.